skims.cfg

```diff
@@ -53,12 +53,11 @@
     options = --outputs TPileupReweighting.prw.root
     dest = SFU-LCG2_LOCALGROUPDISK,
 [mc12_hadhad_ztautau]
```

Comment:
Ok added [ztautau_llskim_test] and [ztautau_lhskim_test] 

does anyone know if it's possible to get a group account on lxplus for HLeptons. I know that HWW store a lot of their ntuples for common usage like that. I set these test blocks to now run from my public area on lxplus (since all the d3pd team are working there) but I really hope this isn't a long term solution.